release notes/v0.36.0.md

```diff
@@ -0,0 +1,135 @@
+
+## Breaking changes
+
+### Restricting file opening to init context
+
+Opening files is now restricted to the init context of the VU that's initialized to get the options from the script (`__VU==0`). While it was somewhat possible to open files *only* in other *VUs* (*e.g* `__VU==2`) in the past, it was [unreliable](https://github.com/grafana/k6/issues/1771). [#2314](https://github.com/grafana/k6/pull/2314) ensures that k6 would now throw an error in a similar scenario. This means that you can still open files only for some VUs, but you need to have opened all of those files in the initial VU (`__VU==0`).
+
+The following statement, unless the `arr.json` file is opened somewhere else, will now throw an error:
+```javascript 
+let file;
+
+if (__VU == 0) {
+  open("./file1.bin")
+  open("./file2.bin")
+} else if (__VU % 2 == 0) {
+  file = open("./file1.bin")
+} else {
+  file = open("./file2.bin")
+}
+
+export default () => {
+  // use file for something
+}
+```
+
+## Bugs Fixed!
+
+
+* We addressed an issue uncovered by [our community](https://community.k6.io/t/v0-35-0-grpc-server-reflection-error/2383), which kept our users from using GRPC with multiple services definition in a single *proto* file. This issue was solved in [#2265](https://github.com/grafana/k6/pull/2265).
+* Thanks to the contribution of [@Resousse](https://github.com/Resousse), we've now updated k6's [`go-ntlmssp`](https://github.com/Azure/go-ntlmssp) dependency. The updating PR [#2290](https://github.com/grafana/k6/pull/2290) indeed fixes issues with NTLM Authentication backends returning two authorization headers. 
+
+## Maintenance
+
+- We have refactored our implementation of the RampingVU executor, for better clarity and maintainability. See [#2155](https://github.com/grafana/k6/pull/2155).
+- [#2316](https://github.com/grafana/k6/pull/2316) relaxed quite a few of the code linting rules we applied to k6's code.
+- [#2304](https://github.com/grafana/k6/pull/2304) prepared the removal of external dependencies from k6's JSONAPI compliant Rest API, and deprecated the `api.v1`'s `client.Call` method in favor of its newer `client.CallAPI` counterpart. It allows us to both reduce our reliance on external dependencies and improve its maintainability.
+- We have updated our Goja dependency, our JS interpreter, to its latest available version. We are, unfortunately, not ready to integrate some of the new features it supports just yet. Namely, Goja now supports the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) syntax, but the Babel version we use presently does not. See [#2317](https://github.com/grafana/k6/pull/2317) and [#2238](https://github.com/grafana/k6/pull/2238)
+- [#2312](https://github.com/grafana/k6/pull/2312) upgraded `loadimpact/k6` docker image base to Alpine *3.15*.
```

Comment:
```suggestion
- [#2312](https://github.com/grafana/k6/pull/2312) upgraded [loadimpact/k6](https://hub.docker.com/r/loadimpact/k6) docker image base to Alpine *3.15*.
```

Reply:
+ isn't it a policy to add usernames in these cases? Others are mentioned it seems...